### PR TITLE
Add a Definition for AU to Configure Deployment Resources

### LINKED ISSUE #1349: Add a Definition for AU to Configure Deployment Resources
Slack conversation here: https://astronomer.slack.com/archives/C015V2JFKT5/p1665771276853749

1 AU is 0.1 vCPU & 0.25 GB mem

### PR DESCRIPTION
**Pull request by kwcanuck**
Resolves #1349

### CODE REVIEW

**Review by jwitz — CHANGES_REQUESTED** [review]
Looks good, although Im not sure where that 1.88 GiB figure comes from. That's what it shows in the Astro UI, but it's not aligned with what we're documenting here. Can we hold off on merging this until we know where the number comes from? 

I've heard we're trying to get rid of AUs in the UI lately, so I'd also check productboard/ the PMs to see when that plan might take place.

File: astro/configure-deployment-resources.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Worker queues are a set of configurations that apply to a group of workers in yo
 
 ## Scheduler resources
 
-The [Airflow scheduler](https://airflow.apache.org/docs/apache-airflow/stable/concepts/scheduler.html) is responsible for monitoring task execution and triggering downstream tasks when the dependencies are met. By adjusting the **Scheduler Count** slider in the **Configuration** tab of the Cloud UI, you can configure up to 4 schedulers, each of which will be provisioned with the AU specified in **Resources**.
+The [Airflow scheduler](https://airflow.apache.org/docs/apache-airflow/stable/concepts/scheduler.html) is responsible for monitoring task execution and triggering downstream tasks when the dependencies are met. By adjusting the **Scheduler Count** slider in the **Configuration** tab of the Cloud UI, you can configure up to 4 schedulers, each of which will be provisioned with the AU specified in **Resources**. An AU represents the total CPU and memory allocated to each scheduler in a Deployment and 1 AU is equivalent to 0.1 CPU and 0.25 GiB of memory. Assigning 5 AUs to a scheduler is equivalent to 0.5 CPUs and 1.88 GiB of memory. You can view the CPU and memory allocations for schedulers on the Deployment **Details** page in the Cloud UI.
```
[the remark this answers is not in the document]
Comment (kwcanuck) [reply astro/configure-deployment-resources.md:16]: @jwitz the values were taken directly from the UI:

image

[the remark this answers is not in the document]
Comment (jwitz) [reply astro/configure-deployment-resources.md:16]: After playing around in the UI more, it looks like one AU is actually around `0.37` of GiB, though not exact. I updated my suggestion to reflect that

**Review by jwitz — APPROVED** [review]
Approving now that we have a (more) accurate figure for an AUs GiB

**Pull request merged**